app/src/main/res/layout/sensor_main.xml

```diff
@@ -45,7 +45,8 @@
                 android:gravity="center"
                 android:textAlignment="center"
                 android:textSize="@dimen/sensor_body_font"
-                android:textStyle="bold" />
+                android:textStyle="bold"
```

Comment:
Text without bold style will look good.

Reply:
Yes. Please make the text inside textview normal